src/com/palmergames/bukkit/towny/TownyAPI.java

```diff
@@ -216,22 +199,24 @@ public boolean isWilderness(Block block) {
      * @return true if the {@link Location} is in the wilderness, false otherwise.
      */
     public boolean isWilderness(Location location) {
-        WorldCoord worldCoord;
-        
-        try {
-            worldCoord = new WorldCoord(townyUniverse.getDataSource().getWorld(location.getWorld().getName()).getName(), Coord.parseCoord(location));
-        } catch (NotRegisteredException e) {
-            // No record so must be Wilderness
-            return true;
-        }
+        return isWilderness(new WorldCoord(WorldCoord.parseWorldCoord(location)));
```

Comment:
Isn't the `new WorldCoord()` redundant as `WorldCoord.parseWorldCoord(location)` will already return a new world coord object.

Reply:
Good catch.